framework/src/setroubleshoot/browser.py

```diff
@@ -790,12 +790,19 @@ def show_current_alert(self):
         alert.update_derived_template_substitutions()
 
         self.toggles=[]
-        for p, args in plugins:
-               rb = self.add_row(p, alert, args)
 
         if len(plugins) == 1:
+               rb = self.add_row(p, alert, args)
                rb.set_active(True)
                self.on_sev_togglebutton_activated(rb, 1)
+        
+        if len(plugins) > 1:
+               highest_priority = plugins[0][0].priority
+               for p, args in plugins:
+                    if p.priority != highest_priority:
```

Comment:
If highlight was False by default this would be p.priority == highest_priority: and add_row(..., highlight=True).

What do you think about it?